src/WebJobs.Script/Diagnostics/FileLoggerProvider.cs

```diff
@@ -17,7 +19,20 @@ public FileLoggerProvider(ScriptHostConfiguration config, Func<string, LogLevel,
             _filter = filter;
         }
 
-        public ILogger CreateLogger(string categoryName) => new FileLogger(categoryName, _config, _filter);
+        public ILogger CreateLogger(string categoryName)
+        {
+            switch (categoryName)
+            {
+                case ScriptConstants.LogCategoryHostNodeConsoleLogs:
```

Comment:
Yep, I'm not a big fan of what I did to wire this up, happy for the feedback to add flexibility.